docs/faq.md

```diff
@@ -51,6 +51,19 @@ The difference in gene sets can be detected when downloading multiple experiment
 (When aggregating by species to create a matrix comprised of samples from different experiments, genes that are not present in all samples [are dropped](#why-do-i-get-a-limited-number-of-genes-back-when-i-aggregate-samples-from-different-experiments).)
 When this occurs, it is because the experiments were quantified with Salmon transcriptome indices using different Ensembl releases of the same genome build.
 
+#### How can I find out what genome build and release were used to process RNA-seq data?
+
+Genome build and Ensembl release version information is available on the pop-up displayed when you click a sample's processing information link in the sample table on the page for an experiment.
+
+![](images/genome-build-sample-modal.gif)
```

Comment:
If you are going to switch this, you may want to switch the gif in the next section, because now this one is much bigger than that one. 

(I don't know how this will look in the final product on http://docs.refine.bio/ -- is there a way I can see that?)

Reply:
Yes, https://github.com/AlexsLemonade/refinebio-docs#to-install + to run section. I checked this way and they look the same size to me, which is about as good as _I_ can do before this gets merged.

Reply:
I can't get it to install because of python issues, but if they look the same size to you; I trust you.